src/zframe.c

```diff
@@ -40,6 +40,10 @@ struct _zframe_t {
 #ifdef ZMQ_GROUP_MAX_LENGTH
     char group[ZMQ_GROUP_MAX_LENGTH + 1];
 #endif
+#ifdef CZMQ_BUILD_DRAFT_API
```

Comment:
Note that it's not necessary to wrap in ifdef the implementation. The symbols will be hidden from the shared library if the API is draft and the build option is not enabled, so it can't be used anyway. This should make the implementation easier and cleaner.

Reply:
Cool, will fix it. I do want to leave it in the other functions as I don't want to change existing stable logic.

Reply:
Fixed, I also fixed the other method eventually, I think it is pretty safe.